### PR TITLE
Buddha Mind, Universe, and Awakening sheng-yen

### PR DESCRIPTION
**Pull request by JJM317**
This was a toughie. I'm still wondering about monograph, the external link, and page numbers.

### CODE REVIEW

File: _content/monographs/buddha-mind-universe-and-awakening_sheng-yen.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+title: "The Buddha Mind, Universe, and Awakening"
+authors:
+  - sheng-yen
+external_url: "https://www.shengyen.org/freebook/pdf/The_Buddha_Mind,_Universe,_and_Awakening.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1iGVeo31NGVlyyE6vISL_hB3NE6HGTLWO/view?usp=drive_link"
+course: dialogue
+tags:
+  - physics
+  - karma
```
Comment (buddhist-uni) [inline _content/monographs/buddha-mind-universe-and-awakening_sheng-yen.md:11]: Perhaps `origination` instead?

**Review by buddhist-uni — COMMENTED** [review]
Yeah monograph and `publisher: dharmadrum` is fine 😊

File: _content/monographs/buddha-mind-universe-and-awakening_sheng-yen.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  - chan
+  - yogacara
+year: 2009
```
[the remark this answers is not in the document]
Comment (JJM317) [reply _content/monographs/buddha-mind-universe-and-awakening_sheng-yen.md:14]: The talks took place in 2008. Toward the end of the ebook there is a copyright page and it lists July 2009 as the first printing.

[the remark this answers is not in the document]
Comment (buddhist-uni) [reply _content/monographs/buddha-mind-universe-and-awakening_sheng-yen.md:17]: Ah! Eagle-eyed! 😅 Thanks 😊

**Pull request merged**